plugins/a11y-keyboard/msd-a11y-keyboard-atspi.c

```diff
@@ -0,0 +1,158 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 8 -*-
+ *
+ * Copyright (C) 2020 Colomban Wendling <cwendling@hypra.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#include "config.h"
+
+#include "msd-a11y-keyboard-atspi.h"
+
+#include <glib-object.h>
+#include <gdk/gdk.h>
+#include <atspi/atspi.h>
+
+
```

Comment:
IMHO, you don't need to use two consecutive blank lines.

Reply:
nitpicky :)  But yeah we don't have those in other similar files, so I dropped one